test/suite/test_rollback_to_stable13.py

```diff
@@ -183,3 +183,73 @@ def test_rollback_to_stable_with_aborted_updates(self):
         stat_cursor = self.session.open_cursor('statistics:', None, None)
         restored_tombstones = stat_cursor[stat.conn.txn_rts_hs_restore_tombstones][2]
         self.assertEqual(restored_tombstones, nrows)
+
+    def test_rollback_to_stable_with_history_tombstone(self):
+        nrows = 1000
+
+        # Prepare transactions for column store table is not yet supported.
+        if self.prepare and self.key_format == 'r':
```

Comment:
Is it possible to add prepared transactions to this test?

Reply:
There is already some test to describe the same transaction prepared updates in test_prepare08.py cover the scenario.